src/Common/tests/XunitTraitsDiscoverers/PlatformID.cs

```diff
@@ -12,6 +12,7 @@ public enum PlatformID
         Windows = 1,
         Linux = 2,
         OSX = 4,
+        AnyUnix = Linux | OSX,
         Any = Windows | Linux | OSX
```

Comment:
Would it make sense to have `Any` on top with value `~0` (so to depict the true meaning of "any"), or is it intentional to confine the OS set?